docs/site/Querying-data.md

```diff
@@ -0,0 +1,279 @@
+---
+lang: en
+title: 'Querying data'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Querying-data.html
+---
+
+## Overview
+
+A _query_ is a read operation on models that returns a set of data or results.
```

Comment:
Don't they write too?

Reply:
I think the query here means the query we use for the `GET` request, where these filters can be applied. Do you have any suggestions?